src/styles/blocks/card.scss

```
@@ -0,0 +1,100 @@
+.card {
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__rate-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 16px;
+  }
+
+  &__image {
+    display: block;
+
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    width: 166px;
+    height: 36px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $color-main-accent;
+  }
+
+  &__rate-reviews {
+    font-size: 10px;
+    font-weight: 400;
+
+    line-height: 14px;
+    color: $color-main-accent;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+  }
```

I think it would be better if css rules would go in the same order as tags in html file. At least that would make it easier to review)
```suggestion
  &__code {
    margin-bottom: 16px;

    font-size: 10px;
    line-height: 14px;
    color: $color-secondary;
  }
  
  &__rate-reviews {
    font-size: 10px;
    font-weight: 400;

    line-height: 14px;
    color: $color-main-accent;
  }
```

---

fixed